### PR TITLE
Adding/fixing missing Zephyr keys and polishing tests

### PR DESCRIPTION
**Pull request by prapti**
More fixes for missing Zephyr keys:
- `header_spec.js`: `CS-14730` to `MM-T881_1` -- as documented in the test case's Cypress implementation PR
- `at_mentions_spec.js`: Changed group from messaging to notifications to match the test case + some cleanup
- `mixed_spec.js`: cleanup
- `revoke_all_sessions_spec.js`: Added `MM-T940`

### CODE REVIEW

**Review by saturninoabril — APPROVED** [review]
Thanks Prapti!

File: e2e/cypress/integration/messaging/header_spec.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ describe('Header', () => {
         updateAndVerifyChannelHeader('>', 'newheader'.repeat(20));
     });
 
-    it('CS14730 - Channel Header: Markdown quote', () => {
+    it('MM-T881_1 - Header: Markdown quote', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/header_spec.js:39]: I think this needs a follow up PR since the test case has 2 steps.

Comment (prapti) [reply e2e/cypress/integration/messaging/header_spec.js:39]: Yeah, follow up on the second step in a follow up PR.
Thanks Saturn!

**Review by furqanmlk — COMMENTED** [review]
LGTM
image

image

image

**Review by furqanmlk — APPROVED** [review]

**Pull request merged**